src/REopt.jl

```diff
@@ -63,6 +63,9 @@ using Shapefile
 using ArchGDAL
 using Roots: fzero  # for IRR
 global hdl = nothing
+using HDF5, JLD
+using CSV, DataFrames
```

Comment:
I have found DelimitedFiles to be more efficient at reading in CSV files (if you just need the data from the CSV file).  What are we using DataFrames for in the package? What is JLD used for in the package?

Reply:
I believe DataFrames and CSV were only being used in one place: emissions_series() in electricutility.jl. I have converted that function to use DelimitedFiles. will push it shortly.

Not sure about JLD?

Reply:
JLD was from a previous iteration of the code for loading the emissions data. I removed all these packages from the project.

Reply:
We might need to keep JLD and HDF5 as they are being used to load EASIUR data in Financial.jl -> get_EASIUR2005( ), unless we now have a different method of loading EASIUR data?

I found some other remaining modules which I have removed such as PolygonInbounds, Shapefile, and Proj4